coldfront/core/project/templates/project/project_list.html

```diff
@@ -118,10 +118,21 @@ <h1>{{ PROGRAM_NAME_SHORT }} Cluster Projects</h1><hr>
             <i class="fas fa-sort-down"></i>
           </a>
         </th>
+        <th scope="col" class="text-nowrap">
```

Comment:
I get an error when attempting this--do you? I'm fine with not having it sortable.

Reply:
Yeah I removed it